InvenTree/build/serializers.py

```diff
@@ -914,12 +919,15 @@ def save(self):
 
                 try:
                     # Create a new BuildItem to allocate stock
-                    BuildItem.objects.create(
+                    build_item, created = BuildItem.objects.get_or_create(
                         build_line=build_line,
                         stock_item=stock_item,
-                        quantity=quantity,
-                        install_into=output
+                        install_into=output,
                     )
+                    if created:
+                        build_item.quantity = 0
```

Comment:
This is necessary because the quantity is 1 by default, when the build item gets created. Alternatively one could also do something like
```py
if created:
    build_item.quantity = quantity
else:
    build_item.quantity += quantity
```
I guess. Or just set the default to 0, although that might not be desirable.

Reply:
I think your second suggestion (not setting it to zero) makes more sense here

Reply:
Perfect, done!